spec/helpers/application_helper_spec.rb

```diff
@@ -17,9 +17,9 @@
     end
   end
 
-  describe '#paragraph_wrap' do
+  describe '#notes_wrap' do
     it 'separates multivalued fields into paragraphs' do
-      expect(helper.paragraph_wrap(value: %w(a b))).to eq '<p>a</p><p>b</p>'
+      expect(helper.notes_wrap(value: %w(a b))).to eq '<ul class="general-notes"><li>a</li><li>b</li></ul>'
```

Comment:
do you want to add a test here for multi-paragraph notes? my understanding is that the `li` implementation is to accomodate them.

Reply:
ah! good catch. I've updated the test description and added some embedded paragraph tags. Just waiting to appease the Rubocop now...